app/models/claim.rb

```diff
@@ -431,8 +433,24 @@ def same_claimant?(other_claim)
     end
   end
 
+  def awaiting_provider_verification?
+    return false unless has_fe_policy?
+
+    return false if eligibility.verification.present?
```

Comment:
Slight nitpick but could we use
```suggestion
    return false if eligibility.verified?
```
so we keep the definition of "being verified" in one place?